watershed-android/watershed/app/src/main/java/com/blueprint/watershed/MiniSites/MiniSiteFragment.java

```diff
@@ -136,13 +136,13 @@ public void onResume() {
     }
 
     @Override
-    public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
-        // Load Field Report
-        FieldReport fieldReport = getFieldReport(position);
-        AddFieldReportFragment addFieldReportFragment = new AddFieldReportFragment();
-        addFieldReportFragment.configureWithFieldReport(fieldReport);
-        mParentActivity.replaceFragment(addFieldReportFragment);
-    }
+    public void onItemClick(AdapterView<?> parent, View view, int position, long id) {}
+//        // Load Field Report
+//        FieldReport fieldReport = getFieldReport(position);
+//        AddFieldReportFragment addFieldReportFragment = new AddFieldReportFragment();
+//        addFieldReportFragment.configureWithFieldReport(fieldReport);
+//        mParentActivity.replaceFragment(addFieldReportFragment);
+//    }
```

Comment:
Won't this make the user unable to click into field reports from the mini site view?